validator/client/attest_protect.go

```diff
@@ -63,18 +63,9 @@ func (v *validator) slashableAttestationCheck(
 	if err != nil {
 		return errors.Wrapf(err, "could not mark epoch %d as attested", indexedAtt.Data.Target.Epoch)
 	}
-	if err := v.db.SaveAttestationHistoryForPubKeyV2(ctx, pubKey, newHistory); err != nil {
+	if err := v.db.SaveAttestationHistoryForPubKeyV2(ctx, pubKey, newHistory, indexedAtt.Data.Source.Epoch, indexedAtt.Data.Target.Epoch); err != nil {
```

Comment:
Can you just pass the attestation? Right now, this API would be really easy to misuse (accidentally switched target and source but their both uint64).

Reply:
sure